packages/ove-asset-manager/README.md

```diff
@@ -1,7 +1,112 @@
-To run this Service you must add Object store credentials into the Appsettings.json file as follows
+# OVE Asset Manager Services
 
-  "s3Client": {
-    "AccessKey": "secret",
+This collection of microservices provides end users with a simple way of managing and processing content to be displayed using OVE. 
+
+## Asset Manager
+
+The core service is the **Asset Manager**. This enables media and content to be uploaded to an **S3** compatible **object store** and records metadata about its processing in an **SQL database**. A full user friendly website is provided to interact with the Asset Manager. A full RESTful API is provided for advanced users.  
+
+## Asset Services
+
+A variety of other Asset Services are provided to process assets into a form advantageous for display using OVE. These services are generally headless and will automatically register themselves with an Asset Manager via its API. They will then generally automatically process assets which require their services. For example the [**Image Tile Service**](https://github.com/ove/ove-asset-services/blob/master/packages/ove-service-imagetiles/README.MD) will create a Deep Zoom image (.dzi file) and corresponding tileset from each image asset. 
+
+## Scalability
+
+The system is intended to be scalable and many copies of the Asset Manager can be run - the EntityFramework and SQL database concurrency models ensure that the ``ProcessingState`` for each Asset is atomically updated despite running multiple instances. 
+
+The Asset Processing Microservices work on a pull based model - they will each independently regularly request further work. You are encouraged to run multiple copies of each Asset Processing Microservice: these should be configured to communicate with an instance of the Asset Manager. These services will then periodically check for Assets to process. Your database and object store may each scale independently. 

```

Comment:
Do they poll for work at a regular interval (if so, what?) or just announce when they have finished a task?

Reply:
They poll regularly for new work at an interval configurable through the appsettings.json. I'll add doc for this.